src/convexhull.jl

```diff
@@ -0,0 +1,20 @@
+function upperhull(pset::PointSet) 
+
+  xs=Vector{Float64}[] 
+  xs=sort(coordinates.(pset), by=first)
+
+  convHull=[xs[1],xs[2]] 
+  
+  for i in 3:size(xs,1)
+    push!(convHull,xs[i])
+
+    # remove the second to last point in convHull while the last 3 points make a "left turn"
+    while size(convHull)[1]>2 && ∠(Point(xs[i-2]),Point(xs[i-1]),Point(xs[i]))<0 
+      splice!(convHull,i-1)
+    end 
+
+  end
+
+  convHull
```

Comment:
I need to review the algorithm, but isn't this code only returning the upper part of the hull? Can you choose a better variable name?